src/templates/static/home.jsx

```diff
@@ -310,20 +310,39 @@ const Home = () => {
             </div>
 
             <div className='gr-padding-10 fill-bg-color'>
-                <div className='container gr-padding-20'>
-                    <div className='gr-row'>
-                        <div className='gr-3 gr-12-m align-self-center center-text-m gr-padding-10'>
+                <div className='gr-padding-20'>
+                    <div className='gr-row container'>
+                        <div className='gr-12 align-self-center center-text gr-padding-10'>
                             <h3 className='no-margin'>{it.L('Award-winning trading excellence')}</h3>
                         </div>
-                        <div className='gr-9 gr-12-m align-self-center'>
-                            <div className='gr-row gr-row-align-center'>
-                                <div className='gr-1 gr-hide-m border-right' />
-                                <div className='gr-2 gr-4-m align-self-center gr-padding-10'><img className='responsive' src={it.url_for('images/pages/home/awards/tw2.svg')} /></div>
-                                <div className='gr-2 gr-4-m align-self-center gr-padding-10'><img className='responsive' src={it.url_for('images/pages/home/awards/gbaf.svg')} /></div>
-                                <div className='gr-3 gr-4-m align-self-center gr-padding-10'><img className='responsive' src={it.url_for('images/pages/home/awards/opwa.svg')} /></div>
-                                <div className='gr-2 gr-4-m align-self-center gr-padding-10'><img className='responsive' src={it.url_for('images/pages/home/awards/menafxpro.svg')} /></div>
-                                <div className='gr-2 gr-4-m align-self-center gr-padding-10'><img className='responsive' src={it.url_for('images/pages/home/awards/egr.svg')} /></div>
-                                <div className='gr-1 gr-hide-m' />
+                        <div className='award-icons'>
+                            <div className='award-icon'>
+                                <img className='responsive' src={it.url_for('images/pages/home/awards/t2w-2010.svg')} />
```

Comment:
Should we start adding `alt` tags to images for SEO and accessibility?

Reply:
Yes, let's add the `alt` tags

Reply:
Added alt tags.